data/src/normalize.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+
+from tempfile import TemporaryFile
+from unicodedata import normalize
+
+'''
+normalize.py
+
+Takes a file and normalizes it "in place." In order to avoid the issues of
+reading and writing to the same file at the same time, this script puts the 
+normalized version of the file argument in a tempfile, then uses that tempfile
+to rewrite the original file.
+'''
+
+def main(args: argparse.Namespace) -> None:
```

Comment:
There is way too much whitespace in this little function. Use a four-space indent (as is the fashion) and get rid of the blank lines. I think blank lines in a single function make the function boundary very hard to discern. If you're tempted to paragraph-ify things, put a comment in instead, or write shorter functions.